_posts/2021-12-01-plugins-intro.markdown

```diff
@@ -106,7 +106,7 @@ Java applications are prone to have vulnerabilities on a remote cluster or by a
 ### How Security Manager is used in OpenSearch?
 
 * As OpenSearch bundles a few plugins, every plugin can define its own custom security policy file which will be installed at the same time when OpenSearch is installing the plugin.
-* Security Manager is initialized in [Opensearch.java](https://github.com/opensearch-project/OpenSearch/blob/1.2/server/src/main/java/org/opensearch/bootstrap/OpenSearch.java#L91) and every plugin has a custom policy file called [plugin-security.policy](https://github.com/opensearch-project/anomaly-detection/blob/main/src/main/plugin-metadata/plugin-security.policy).
+* Security Manager is initialized in [Opensearch.java](https://github.com/opensearch-project/OpenSearch/blob/1.2/server/src/main/java/org/opensearch/bootstrap/OpenSearch.java#L91) and the plugins that require security polices has a custom policy file called [plugin-security.policy](https://github.com/opensearch-project/anomaly-detection/blob/main/src/main/plugin-metadata/plugin-security.policy).
```

Comment:
Small suggestion. Add a comma after the link closing parenthesis and change **has** to **have**. Like below:

"), and the plugins that require security policies has->have a custom policy file"

Reply:
Thanks for the review @elfisher. Pushed the change.